diff.py

```diff
@@ -2129,8 +2133,12 @@ class Diff:
     max_score: int
 
 
-def is_nops(lines: List[Line]) -> bool:
-    return all(line.mnemonic == "nop" for line in lines)
+def get_trim_point(lines: List[Line], arch_settings: ArchSettings) -> bool:
+    for index, line in enumerate(reversed(lines)):
+        rindex = len(lines) - index - 1
```

Comment:
```suggestion
    for rindex, line in reversed(enumerate(lines)):
```
(and rename rindex -> index)